glotaran/builtin/models/kinetic_spectrum/spectral_constraints.py

```diff
@@ -49,7 +53,7 @@ def applies(interval):
 @model_attribute(
     properties={
         "compartment": str,
-        "interval": typing.List[typing.Tuple[float, float]],
+        "interval": List[Tuple[float, float]],
```

Comment:
If that type is correct, than there is no point in checking
```python
if isinstance(self.interval, tuple):
```
in the implementation.
Else the type should be `Union[Tuple[float, float], List[Tuple[float, float]]]`.